assets/test/specs/form.spec.js

```diff
@@ -220,5 +229,41 @@ describe('Form Validation', function () {
       });
     });
 
+    describe('When I toggle open the grouped input elements, do not select an item in the group and submit', function () {
+      beforeEach(function () {
+        $groupToggle.click();
+
+        $submitButton.click();
+      });
+
+      it('The error should be visible', function () {
+        expect($groupToggle.closest('.form-field-group')).toHaveClass('form-field-group--error');
+      });
+
+
+      it('The form should have the grouped radio required error message', function () {
+        var $errorMessages = $errorSummaryMessages.find('> li');
+        expect($errorMessages.text().indexOf($radioYesGroupElement.attr('data-msg-required'))).not.toBe(-1);
+      });
+    });
+
+    describe('When I toggle open the grouped input elements, select an item and toggle closed the group and submit', function () {
+
+      beforeEach(function () {
+        $submitButton.click();
+      });
+
+      it('The error should not be visible', function () {
+        expect($groupToggle.closest('.form-field-group')).not.toHaveClass('form-field-group--error');
+      });
+
+      it('The form should not have the grouped radio required error message', function () {
+        var $errorMessages = $errorSummaryMessages.find('> li');
+
+        for (var e = 0; e < $errorMessages.length; e++) {
+          expect($errorMessages.eq(e).text()).not.toBe($radioNoGroupElement.attr('data-msg-required'));
+        }
+      });
+    });
```

Comment:
I would also add a further describe
describe('When I open the grouped input elements, hit submit and cause an error, then close the grouped input elements');
it('The error should not be visible');
it('The form should have the grouped radio required error message');

**Note you will need to add something along the lines of https://github.com/hmrc/assets-frontend/blob/PE-1926-validation-bug/assets/test/specs/form.spec.js#L94-L97 to stop the page refreshing on successful submit**